src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Infrastructure/ApiClients/RoatpApplicationApiClient.cs

```diff
@@ -24,97 +25,149 @@ public async Task<Apply> GetApplication(Guid applicationId)
 
         public async Task<List<AssessorSequence>> GetAssessorSequences(Guid applicationId)
         {
-            var assessorSequences = await Get<List<AssessorSequence>>($"/Assessor/{applicationId}/Overview");
+            return await Get<List<AssessorSequence>>($"/Assessor/Applications/{applicationId}/Overview");
+        }
+
+        public async Task<AssessorPage> GetAssessorPage(Guid applicationId, int sequenceNumber, int sectionNumber, string pageId)
+        {
+            AssessorPage assessorPage;
+
+            if(string.IsNullOrEmpty(pageId))
+            {
+                assessorPage = await Get<AssessorPage>($"/Assessor/Applications/{applicationId}/Sequences/{sequenceNumber}/Sections/{sectionNumber}/Page");
+            }
+            else
+            {
+                assessorPage = await Get<AssessorPage>($"/Assessor/Applications/{applicationId}/Sequences/{sequenceNumber}/Sections/{sectionNumber}/Page/{pageId}");
+            }
+
+            return assessorPage;
+        }
+
+        public async Task SubmitAssessorPageOutcome(Guid applicationId, 
```

Comment:
Would it be useful to return a success flag here? For example, checking the HTTP code of the response from the service

Reply:
Returned HttpStatusCode is logged now.

Reply:
Logging is fine, but the method should return some indication of success/failure. For example, if the submit outcome call returns a 500 error, the code will currently carry on as if it worked OK

Reply:
I will return the response to the calling method and then what? 
In case of a failure, shall I display error message to the user?
Send email notification to the admin? 
Logging it there as well?
We don’t have anything in the acceptance criteria covering this.
What is the point if returning it, if we don’t use it?
And as far as I know in similar scenarios, we don’t do anything.